virttest/utils_misc.py

```diff
@@ -2755,14 +2755,26 @@ def format_linux_disk(session, did, mountpoint, size, fstype="ext3"):
     for disk in disks.splitlines():
         if did in disk:
             kname = disk.split()[0]
+            list_parttable_cmd = "parted -s %s print" % kname
+            output = session.cmd_output(list_parttable_cmd)
+            if "Error" in output:
+                logging.info("Create partition table on disk '%s'" % kname)
+                mklabel_cmd = "parted -s %s mklabel msdos" % kname
```

Comment:
I don't understand "parted /tmp/zz.qcow2 -s "mklabel msdos"" ,do you mean can creat partition table to image file directly? I test it ，and I can't boot up my vm with error “Image is not in qcow2 format
”